python/geom.py

```diff
@@ -436,10 +475,14 @@ def __init__(self, vertices, height, axis=Vector3(z=1), center=None, **kwargs):
 
 class Matrix(object):
 
-    def __init__(self, c1=Vector3(), c2=Vector3(), c3=Vector3()):
+    def __init__(self, c1=Vector3(), c2=Vector3(), c3=Vector3(), diag=Vector3(), offdiag=Vector3()):
         self.c1 = c1
         self.c2 = c2
         self.c3 = c3
+        if c1 == c2 == c3 == Vector3():
+            self.c1 = Vector3(diag.x,offdiag.x,offdiag.y)
+            self.c2 = Vector3(offdiag.x,diag.y,offdiag.z)
+            self.c3 = Vector3(offdiag.y,offdiag.z,diag.z)
```

Comment:
The `offdiag` parts represent a complex Hermitian matrix in general, so you should take the conjugate of the lower-triangular part of the matrix with `np.conj` (which works for both real and complex values).

Reply:
good catch -- fixed with latest commit.